src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -1,3 +1,6 @@
+<cd-alert-panel *ngIf="isTimeOut"
+                type="warning"
```

Comment:
Maybe use the slim version of the component
```suggestion
                type="warning"
                size="slim"
```

Reply:
And in ngIf need to write "isStale" ,right?